src/api/python/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin" AND
   # versions of CMake. If we are compiling cvc5 on macOS and we have a newer
   # version of CMake available, we try to use find_package(Python ...) to find
   # the paths.
-  find_package(Python COMPONENTS Development)
+  find_package(Python 3.10 EXACT COMPONENTS Development)
```

Comment:
cvc5 supports Python 3.6 up to 3.10, so we should not force our users to use an exact version here. We could do a range check here `3.6...<3.11` instead of 3.10 and add a comment why we exclude 3.11 for now.

Reply:
Fixed.